sentry-ruby/spec/sentry/hub_spec.rb

```diff
@@ -222,9 +222,12 @@
       expect(event[:monitor_config]).to include({ schedule: { type: :crontab, value: "* * * * *" } })
     end
 
-    it_behaves_like "capture_helper" do
-      let(:capture_helper) { :capture_check_in }
-      let(:capture_subject) { [slug, :ok] }
+    it "doesn't send the event nor assign last_event_id" do
```

Comment:
Let's wrap this inside a context like `"with sending not allowed"` so it doesn't look like `capture_check_in` should not be sent.